<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testng-results skipped="0" failed="1" ignored="101" total="102" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="5198" started-at="2019-03-01T13:10:35Z" finished-at="2019-03-01T13:10:40Z">
    <groups>
      <group name="Anoop_EOD1">
        <method signature="QCStore.EODOnDueDate()[pri:412, instance:tests.QCStore@799f10e1]" name="EODOnDueDate" class="tests.QCStore"/>
      </group> <!-- Anoop_EOD1 -->
    </groups>
    <test name="Test" duration-ms="5198" started-at="2019-03-01T13:10:35Z" finished-at="2019-03-01T13:10:40Z">
      <class name="tests.QCStore">
        <test-method status="PASS" signature="setup()[pri:0, instance:tests.QCStore@799f10e1]" name="setup" is-config="true" duration-ms="105" started-at="2019-03-01T13:10:35Z" finished-at="2019-03-01T13:10:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="killProcess()[pri:0, instance:tests.QCStore@799f10e1]" name="killProcess" is-config="true" duration-ms="5024" started-at="2019-03-01T13:10:35Z" finished-at="2019-03-01T13:10:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killProcess -->
        <test-method status="FAIL" signature="EODOnDueDate()[pri:412, instance:tests.QCStore@799f10e1]" name="EODOnDueDate" duration-ms="23" started-at="2019-03-01T13:10:40Z" finished-at="2019-03-01T13:10:40Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[D:\QC_Batch\QC_LOC\TestData\QCStore\ALOC\QC_LOC_EODDuedate.xls (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: D:\QC_Batch\QC_LOC\TestData\QCStore\ALOC\QC_LOC_EODDuedate.xls (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at tests.ExcelNew.<init>(ExcelNew.java:35)
at tests.QCStore.EODOnDueDate(QCStore.java:5634)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EODOnDueDate -->
        <test-method status="FAIL" signature="getResult(org.testng.ITestResult)[pri:0, instance:tests.QCStore@799f10e1]" name="getResult" is-config="true" duration-ms="10" started-at="2019-03-01T13:10:40Z" finished-at="2019-03-01T13:10:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=EODOnDueDate status=FAILURE method=QCStore.EODOnDueDate()[pri:412, instance:tests.QCStore@799f10e1] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at tests.QCStore.getScreenhot(QCStore.java:7091)
at tests.QCStore.getResult(QCStore.java:7107)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
=======
<testng-results skipped="0" failed="0" ignored="113" total="114" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="1524740" started-at="2019-03-01T11:34:06Z" finished-at="2019-03-01T11:59:31Z">
    <groups>
      <group name="Janaki_EOD">
        <method signature="QCStore.JQC_LOC_Refinance_EOD_Payment_Txn()[pri:323, instance:tests.QCStore@527740a2]" name="JQC_LOC_Refinance_EOD_Payment_Txn" class="tests.QCStore"/>
      </group> <!-- Janaki_EOD -->
    </groups>
    <test name="Test" duration-ms="1524740" started-at="2019-03-01T11:34:06Z" finished-at="2019-03-01T11:59:31Z">
      <class name="tests.QCStore">
        <test-method status="PASS" signature="setup()[pri:0, instance:tests.QCStore@527740a2]" name="setup" is-config="true" duration-ms="90" started-at="2019-03-01T11:34:06Z" finished-at="2019-03-01T11:34:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="killProcess()[pri:0, instance:tests.QCStore@527740a2]" name="killProcess" is-config="true" duration-ms="5003" started-at="2019-03-01T11:34:06Z" finished-at="2019-03-01T11:34:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killProcess -->
        <test-method status="PASS" signature="JQC_LOC_Refinance_EOD_Payment_Txn()[pri:323, instance:tests.QCStore@527740a2]" name="JQC_LOC_Refinance_EOD_Payment_Txn" duration-ms="1519420" started-at="2019-03-01T11:34:11Z" finished-at="2019-03-01T11:59:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- JQC_LOC_Refinance_EOD_Payment_Txn -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:tests.QCStore@527740a2]" name="getResult" is-config="true" duration-ms="210" started-at="2019-03-01T11:59:31Z" finished-at="2019-03-01T11:59:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=JQC_LOC_Refinance_EOD_Payment_Txn status=SUCCESS method=QCStore.JQC_LOC_Refinance_EOD_Payment_Txn()[pri:323, instance:tests.QCStore@527740a2] output={null}]]]>
              </value>
            </param>
          </params>
>>>>>>> feb1.1
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- tests.QCStore -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
